docs/changelog.md

```diff
@@ -16,6 +16,9 @@ what we publish.
 
 ### ⭐️ New
 
+- `List[T]` values are now equality comparable with `==` and `!=` when `T` is
```

Comment:
Don't forget to credit yourself!  See the parts below in the changelog like

```
([PR 2910#](https://github.com/modularml/mojo/pull/2910) by
  [@msaelices](https://github.com/msaelices))
```

for example.

Reply:
One second.

Reply:
Done.